### ISSUE TITLE
Radial menu fror favourite label selection

### ISSUE THREAD

**Issue by mzur** [biigle/core#815]
Here is an idea that could improve label selection for users working with a tablet and pen: In the annotation tool settings, they can enable the "radial label menu". When enabled, they can do a right click (with a button on the pen) that opens a menu like this: 

![image](https://github.com/biigle/core/assets/2457311/804b941c-4d9b-42ff-92fb-ca849993db79)

The menu does not show action buttons as in the screenshot but all favourite labels. The menu is displayed as long as the pen button is pressed. The user then can move the pen to the label they want to select and when they release the button, the label is selected.

**Comment by dlangenk** [comment]
Would also be nice to have a delete last label item in there.

**Comment by mzur** [comment]
This would mix "general" actions with the label selection again. Is it too cumbersome to click on the button at the bottom of the screen?

**Comment by dlangenk** [comment]
The thing is that you slip or the tablet is detecting your hand as input and so an annotation is created which is incomplete and a new one is started. This happens frequently so you quickly want to get rid of it. Thats why I programmed one of the pen buttons to simulate the backspace key.

**Comment by dlangenk** [comment]
The radial menu might even be nice with a mouse. You press mouse button 4 or 5 or a keyboard key and while pressed the radial menu is shown.

**Comment by mzur** [comment]
But I imagine that it could be difficult to show more than 10 items (i.e. the favourite labels) in the menu.

**Comment by dlangenk** [comment]
Yes, but it might be faster than hotkeys (including memorizing the mapping). But that is just a nice to have if you get it for free.
